scene/2d/line_builder.cpp

```diff
@@ -136,21 +137,40 @@ void LineBuilder::build() {
 	Vector2 pos1 = points[1];
 	Vector2 f0 = (pos1 - pos0).normalized();
 	Vector2 u0 = rotate90(f0);
-	Vector2 pos_up0 = pos0 + u0 * hw;
-	Vector2 pos_down0 = pos0 - u0 * hw;
+	Vector2 pos_up0 = pos0;
+	Vector2 pos_down0 = pos0;
 
 	Color color0;
 	Color color1;
 
 	float current_distance0 = 0.f;
 	float current_distance1 = 0.f;
 	float total_distance = 0.f;
+	float width_factor = 1.f;
 	_interpolate_color = gradient != NULL;
+	_retrieve_curve = curve != NULL;
```

Comment:
`_retrieve_curve` doesn't need to be a member boolean, can't it just be a local?

Reply:
Done.
It was in case I need this value in other functions just like _interpolate_color.